RasimKhusaenov/my_own_blog
# Redesign articles list

### Summary

- Changed application offset;
- Redesigned articles list (Now it consists of two columns);
- Added comments count to article card;
- Added more space between "Search" and "New" buttons;
- Fixed bug with new comment field height;
- Change items number on page to 10.

### How it works

![image](https://user-images.githubusercontent.com/35406008/181952784-c5a50e14-9944-43f5-a4fa-82237cb29494.png)

### Test plan

List of steps to manually test introduced functionality:

- Log in
- Go to https://my-own-blog-pr-78.herokuapp.com/
- Make sure that you like it :)

### Deploy notes

Notes regarding deployment the contained body of work.
These should note any db migrations, ENV variables, services, scripts, etc.

### Checklist:


### References
N/A
